### PR TITLE
WW-364 inputs styling

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

File: extensions/wikia/CreateNewWiki/js/WikiBuilder.js [hunk 1]
```diff
@@ -79,13 +80,14 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
 		$themWikiWrapper = $('#ThemeWiki');
 		steps = wb.find('.steps .step');
 		wikiName = $nameWikiWrapper.find('input[name=wiki-name]');
+		wikiNameLabel = $nameWikiWrapper.find('label[for=wiki-name]');
 		wikiNameStatus = $nameWikiWrapper.find('.wiki-name-status-icon');
```
Comment (WikiaTech) [inline extensions/wikia/CreateNewWiki/js/WikiBuilder.js]: MAJOR Remove this useless assignment to local variable "wikiNameStatus" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

File: extensions/wikia/CreateNewWiki/js/WikiBuilder.js [hunk 2]
```diff
@@ -79,13 +80,14 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
...
 		wikiNameError = $nameWikiWrapper.find('.wiki-name-error');
 		wikiDomain = $nameWikiWrapper.find('input[name=wiki-domain]');
+		wikiDomainLabel = $nameWikiWrapper.find('label[for=wiki-domain]');
 		wikiDomainError = $nameWikiWrapper.find('.wiki-domain-error');
 		wikiDomainStatus = $nameWikiWrapper.find('.domain-status-icon');
```
Comment (WikiaTech) [inline extensions/wikia/CreateNewWiki/js/WikiBuilder.js]: MAJOR Remove this useless assignment to local variable "wikiDomainStatus" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues
* MAJOR 7 major

Watch the comments in this conversation to review them.

#### 5 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/c913cc8c1239bf3d92b08646dafa5166b43f1abc/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/c913cc8c1239bf3d92b08646dafa5166b43f1abc/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L459](https://github.com/Wikia/app/blob/c913cc8c1239bf3d92b08646dafa5166b43f1abc/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L459): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. MAJOR [WikiBuilder.js#L593](https://github.com/Wikia/app/blob/c913cc8c1239bf3d92b08646dafa5166b43f1abc/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L593): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L595](https://github.com/Wikia/app/blob/c913cc8c1239bf3d92b08646dafa5166b43f1abc/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L595): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues
* MAJOR 7 major

Watch the comments in this conversation to review them.

#### 5 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/cf0a3c240f576774057122f74e8b7284c0a6dac1/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/cf0a3c240f576774057122f74e8b7284c0a6dac1/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L459](https://github.com/Wikia/app/blob/cf0a3c240f576774057122f74e8b7284c0a6dac1/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L459): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. MAJOR [WikiBuilder.js#L593](https://github.com/Wikia/app/blob/cf0a3c240f576774057122f74e8b7284c0a6dac1/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L593): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L595](https://github.com/Wikia/app/blob/cf0a3c240f576774057122f74e8b7284c0a6dac1/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L595): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/1377090f940bcbae751dae4ba5a89785ccbac526/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/1377090f940bcbae751dae4ba5a89785ccbac526/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L455](https://github.com/Wikia/app/blob/1377090f940bcbae751dae4ba5a89785ccbac526/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L455): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. MAJOR [WikiBuilder.js#L589](https://github.com/Wikia/app/blob/1377090f940bcbae751dae4ba5a89785ccbac526/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L589): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L591](https://github.com/Wikia/app/blob/1377090f940bcbae751dae4ba5a89785ccbac526/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L591): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 3]
```diff
@@ -112,12 +125,10 @@
...
 	.wiki-domain-container {
 		position: relative;
+	}
 
-		img {
-			bottom: 4px;
-			left: -40px;
-			position: absolute;
-		}
+	.wiki-name-container {
```
Comment (rybmat) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: could this one and the one above be a single rule?

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 4]
```diff
@@ -203,6 +214,25 @@
...
 	}
+
+	.floating-label {
+		color: $wds-color-light-gray;
+		left: 0;
+		pointer-events: none;
+		position: absolute;
+		top: 0;
+		transition: font-size ease-in 0.1s,top ease-in 0.1s;
+		-webkit-transition: font-size ease-in 0.1s,top ease-in 0.1s;
```
Comment (rybmat) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: what about transition mixin? https://github.com/Wikia/app/blob/dev/skins/shared/mixins/transition.scss

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/4f72255fb11be3f842b334bf1280d49a2495113d/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
2. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/4f72255fb11be3f842b334bf1280d49a2495113d/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
3. MAJOR [WikiBuilder.js#L455](https://github.com/Wikia/app/blob/4f72255fb11be3f842b334bf1280d49a2495113d/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L455): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
4. MAJOR [WikiBuilder.js#L589](https://github.com/Wikia/app/blob/4f72255fb11be3f842b334bf1280d49a2495113d/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L589): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
5. MAJOR [WikiBuilder.js#L591](https://github.com/Wikia/app/blob/4f72255fb11be3f842b334bf1280d49a2495113d/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L591): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Review by rybmat — APPROVED** [review]

**Review by cormorando — APPROVED** [review]

**Pull request merged**